resources/views/maps/markers/details.blade.php

```diff
@@ -33,6 +33,15 @@
         @endif
     @endif
 
+    @if($marker->entity && $marker->entity->typeID() === config('entities.ids.location') && !$marker->entity->child->maps->isEmpty())
+        <div class="profile-maps">
```

Comment:
For consistency with the action buttons below, add `text-center` as a class to the div (or is it already applied by `profile-maps`?)